scripts/deployment-testnet/deploy_parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "buidlerNetwork": "rinkeby",
+  "4": {
+    "libVerifiersAddress": ["0x86AAD337f0EcD4F81e5B770Ad84BCa505f1B1027"],
+    "libposeidonsAddress": ["0x5bc822aD044dfbD8C54a02f3aab2317CE588c3D6", "0xB4D18800FD0e394d43f60792F447073DC5802176", "0xfE7dcf6bC1b9139F82c9a871884e0cdd9b418e80"],
+    "libVerifiersWithdrawAddress": "0x8A4027F0f2EF9B29b00e7d8Ce5419949C1c5824A",
+    "hermezGovernanceAddress": "0x9CEC5F3d5d9747DCCB36d8e22c03aF8221445c0D",
+    "emergencyCouncilAddress": "0x82AA0Fcb77B6CF3E8baC5994043De98687B95E8E",
+    "donationAddress": "0x929297764aB2cb291C5DbcC1A8D2a0867CafdBDe",
+    "bootCoordinatorAddress": "0x601f4005b90Db8f6Bbf06555f0b20Cf60798EBD8",
+    "genesisBlock": "",
+    "genesisBlockOffsetCurrent": 81,
```

Comment:
Could we change this default parameter to 10 blocks ?
Just to not wait too much when running on testnet

Reply:
nope, 80 is the minimum! we should change the Smart contract for that!